Sources/GRPC/HTTP1ToRawGRPCServerCodec.swift

```diff
@@ -267,14 +267,19 @@ extension HTTP1ToRawGRPCServerCodec: ChannelOutboundHandler {
         // Store the response into an independent buffer. We can't return the message directly as
         // it needs to be aggregated with all the responses plus the trailers, in order to have
         // the base64 response properly encoded in a single byte stream.
-        messageWriter.write(messageBytes, into: &self.responseTextBuffer)
+        //
+        // We can try! here because the server does not support compression at the moment and this
+        // only throws when compression fails.
+        try! messageWriter.write(messageBytes, into: &self.responseTextBuffer)
```

Comment:
Just a note: I'll add support for the server after this PR is wrapped up, the `try!` will disappear then.